person_methods.rb

```diff
@@ -0,0 +1,38 @@
+require './student'
+require './teacher'
+require './wrong_option'
+
+class CreatePerson
+  def make_person
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp
+
+    [age, name]
+    # return CreateStudent.new.create_student(age.to_i, name) if opt == '1'
+    # return create_teacher(age.to_i, name) if opt == '2'
```

Comment:
- [ ] I would recommend you remove commented code as they make your codebase look ugly and messy.